packages/server/src/api/routes/script.ts

```diff
@@ -1,10 +0,0 @@
-import Router from "@koa/router"
```

Comment:
I'm assuming this was unused?

Reply:
Yeap - theres no API that uses this and it didn't even really function properly/was even tested - it didn't serve a purpose and probably never would!